blueprints/node-webkit/files/tests/helpers/qunit-logger.js

```diff
@@ -0,0 +1,62 @@
+/* global QUnit */
+/* global require */
+// dead stupid script to format test output from nw.js to the console
+(function() {
+if(!window._phantom){
```

Comment:
Rather than check for `window._phantom`, I'd suggest checking for `window.nwDispatcher` (which would indicate that you're running this in NW.js). This would ensure calls like `require('nw.gui')` won't throw an error if not in NW.js.